src/applications/vaos/services/appointment/index.js

```diff
@@ -49,7 +56,10 @@ export async function getBookedAppointments({ startDate, endDate }) {
  */
 export async function getAppointmentRequests({ startDate, endDate }) {
   try {
-    const appointments = await getPendingAppointments(startDate, endDate);
+    const appointments = (await getPendingAppointments(
+      startDate,
+      endDate,
+    )).filter(appt => ['Submitted', 'Cancelled'].includes(appt.status));
```

Comment:
What's this extra filtering for?

Reply:
Yeah, filterRequests() should be handling this instead.  Removing the filtering here, and updating the `getAppointmentStatus()` to handle statuses other than 'Submitted' and 'Cancelled'.